test/test-utils-get-limit.js

```diff
@@ -0,0 +1,22 @@
+var getLimit = require('../lib/utils').getLimit;
+
+var assert = require('assert');
+
+assert.deepEqual(getLimit(undefined, 'fieldSize', 1), 1);
```

Comment:
`strictEqual` seems like the more appropriate choice for most of these?

Reply:
Agreed, here and below.